registry/custom/custom.go

```diff
@@ -46,13 +46,6 @@ func customRoutes(cfg *config.Custom, ch chan string) {
 	for {
 		log.Printf("[DEBUG] Custom Registry starting request %s \n", time.Now())
 		resp, err := client.Do(req)
-		if resp != nil {
-			defer func() {
-				if err := resp.Body.Close(); err != nil {
-					log.Printf("Error Can not close HTTP resp body - %s -%s \n", URL, err.Error())
-				}
-			}()
-		}
```

Comment:
Why remove this defer?

Reply:
this one is easy, because its in an infinite loop, the function never exist so the defer is never triggered. that being said maybe I should get the loop to call a sub function that then exits properly.

Reply:
resolved in the follow up commit.